src/EnergyPlus/WeatherManager.hh

```diff
@@ -774,95 +774,98 @@ struct WeatherManagerData : BaseGlobalStruct
-    Real64 SnowGndRefModifier;                  // Modifier to ground reflectance during snow
-    Real64 SnowGndRefModifierForDayltg;         // Modifier to ground reflectance during snow for daylighting
-    Weather::WaterMainsTempCalcMethod WaterMainsTempsMethod; // Water mains temperature calculation method
-    int WaterMainsTempsSchedule;                             // Water mains temperature schedule
-    Real64 WaterMainsTempsAnnualAvgAirTemp;                  // Annual average outdoor air temperature (C)
-    Real64 WaterMainsTempsMaxDiffAirTemp;                    // Maximum difference in monthly average outdoor air temperatures (deltaC)
-    std::string WaterMainsTempsScheduleName;                 // water mains tempeature schedule name
-    bool wthFCGroundTemps;
-
-    int TotRunPers;           // Total number of Run Periods (Weather data) to Setup
-    int TotRunDesPers;        // Total number of Run Design Periods (Weather data) to Setup
-    int TotReportPers;        // Total number of reporting periods
-    int TotThermalReportPers; // Total number of thermal reporting periods
-    int TotCO2ReportPers;     // Total number of CO2 reporting periods
-    int TotVisualReportPers;  // Total number of visual reporting periods
-
-    int NumSpecialDays;
-    Array1D_int SpecialDayTypes; // To hold holiday types given in input file NOLINT(cert-err58-cpp)
-    Array1D_int WeekDayTypes;    // To hold Week day types using specified first day NOLINT(cert-err58-cpp)
-    Array1D_int DSTIndex;        // To hold DST Index based on weather file or input NOLINT(cert-err58-cpp)
-
-    int NumDataPeriods;
-
-    int NumIntervalsPerHour;
-
-    bool UseDaylightSaving;         // True if user says to use Weather File specified DaylightSaving Period
-    bool UseSpecialDays;            // True if user says to use Weather File specified Special Days for current RunPeriod
-    bool UseRainValues;             // True if rain values from weather file are to be used
-    bool UseSnowValues;             // True if snow values from weather file are to be used
-    bool EPWDaylightSaving;         // True if a DaylightSaving Time Period is input (EPW files)
-    bool IDFDaylightSaving;         // True if a DaylightSaving Time Period is input (IDF files)
-    bool DaylightSavingIsActive;    // True if a DaylightSavingPeriod should be used for Environment
-    bool WFAllowsLeapYears;         // True if the Weather File (WF) header has "Yes" for Leap Years
-    int curSimDayForEndOfRunPeriod; // normal=number days in sim, but different when repeating runperiods or multi-year files
-    int Envrn;                      // Counter for environments
-    int NumOfEnvrn;                 // Number of environments to be simulated
-    int NumEPWTypExtSets;           // Number of Typical/Extreme on weather file.
-    int NumWPSkyTemperatures;       // Number of WeatherProperty:SkyTemperature items in input file
+    bool GetBranchInputOneTimeFlag = true;
```

Comment:
Moving all of the inits and ditching the constructor here was a bit overboard, so scrutinize these. Initial values were taken from the constructor (if present there) and from the line-by-line `clear_state` which covered everything.